docker/backend/Dockerfile

```diff
@@ -1,65 +1,55 @@
 FROM hangy/httpd-alpine-perl-modperl AS builder
 
-# Remove APK cache
```

Comment:
I had experienced some issue when this was not in the Dockerfile before, but now it seems to work with `apk -U --no-cache add`. 👍